src/ngx_http_lua_ssl_certby.c

```diff
@@ -1319,6 +1319,21 @@ ngx_http_lua_ffi_set_priv_key(ngx_http_request_t *r,
 }
 
 
+int
+ngx_http_lua_ffi_get_ssl_pointer(ngx_http_request_t *r,
+    ngx_ssl_conn_t **ssl_conn, char **err)
+{
+    if (r->connection == NULL || r->connection->ssl == NULL) {
+        *err = "bad request";
+        return NGX_ERROR;
```

Comment:
It's better to just return the SSL pointer directly and return NULL in case of errors.

Reply:
Will fix.